src/background.c

```diff
@@ -704,6 +710,10 @@ handle_request_background_in_thread_func (GTask *task,
   AutostartFlags autostart_flags = AUTOSTART_FLAGS_NONE;
   gboolean activatable = FALSE;
   g_auto(GStrv) commandline = NULL;
+  const char * const *run_times = { NULL };
```

Comment:
```suggestion
  g_autofree const char * const *run_times = NULL;
```

`^a&s` return like `g_variant_get_strv()` it will leak otherwise.